[2021-07-16 10:18:25,173] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:18:03.176006+00:00 [queued]>
[2021-07-16 10:18:25,212] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:18:03.176006+00:00 [queued]>
[2021-07-16 10:18:25,214] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:18:25,215] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 10:18:25,215] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:18:25,246] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T10:18:03.176006+00:00
[2021-07-16 10:18:25,279] {standard_task_runner.py:52} INFO - Started process 548 to run task
[2021-07-16 10:18:25,288] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T10:18:03.176006+00:00', '--job-id', '510', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpevg8_u_2', '--error-file', '/tmp/tmpgt5jpp9_']
[2021-07-16 10:18:25,290] {standard_task_runner.py:77} INFO - Job 510: Subtask restart_test
[2021-07-16 10:18:25,439] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:18:03.176006+00:00 [running]> on host a2cbcfd4d6f4
[2021-07-16 10:18:25,599] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T10:18:03.176006+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T10:18:00.610369+00:00
[2021-07-16 10:18:29,772] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T101803, start_date=20210716T101825, end_date=20210716T101829
[2021-07-16 10:18:29,841] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 10:18:29,897] {local_task_job.py:151} INFO - Task exited with return code 0
